rct229/rulesets/ashrae9012019/ruleset_functions/g311_exceptions/g311_sub_functions/is_zone_mechanically_cooled.py

```diff
@@ -0,0 +1,88 @@
+import pydash
+from pydash import flat_map, chain, flow, is_equal
```

Comment:
It seems like the ```chain``` and ```is_equal``` functions aren't used.  Also ```import pydash``` isn't used. Could we remove them?